theme/static/js/resourceLandingAjax.js

```diff
@@ -1011,33 +1109,52 @@ function initializeDatePickers(){
 }
 
 // act on spatial coverage type change
-function setFileTypeSpatialCoverageFormFields(){
-    $("#div_id_type_filetype input:radio").change(function () {
-        if ($(this).val() == "point" && $(this).attr("checked") == "checked") {
-            $("#div_id_north_filetype").show();
-            $("#div_id_east_filetype").show();
-            $("#div_id_elevation_filetype").show();
-            $("#div_id_northlimit_filetype").hide();
-            $("#div_id_eastlimit_filetype").hide();
-            $("#div_id_southlimit_filetype").hide();
-            $("#div_id_westlimit_filetype").hide();
-            $("#div_id_uplimit_filetype").hide();
-            $("#div_id_downlimit_filetype").hide();
+function setFileTypeSpatialCoverageFormFields(logical_type){
```

Comment:
@dtarb @gantian127 @pkdash The problem I have is not that the metadata is editable. The problem I have is that it differs from the recorded metadata in the file and there is no clear record of that in resourcemetadata.xml. 

RDF has features that allow that kind of documentation. This process of identifying sources is called "reification". "Reification by entity X" can be understood simply as a statement that "X claims that Y has value Z", rather than the more naïve statement "Y has value Z." Thus, in the metadata, it is recorded that we know that the metadata in the file is one thing, while the metadata the user would like to assert is different. 

The appropriate way to code this into RDF (to DataOne standards) is to put in the recorded metadata as "reified by the file" but also record the edited metadata as "reified by a specific person".  We are not doing either. 

Reification becomes particularly important if we try to record anything crowdsourced about the resource. Then "The HydroShare Community claims that Y has value Z". 

This is not an issue of versioning. It is a matter of how data harvested from the file is presented in the metadata record, and how the metadata record distinguishes between harvested and edited data. 

For example, suppose that someone at CUAHSI categorizes a resource in a particular way. The way to record that is that "CUAHSI thinks that the category for X is Y" rather than "the category for X is Y". 

This is a deep issue. The XML for metadata doesn't contain any reification at this point. 

My main objection is not that the metadata is editable, but that it is substituted for the harvested data without any record in resourcemetadata.xml about that substitution.